sdk/healthinsights/azure-health-insights-radiologyinsights/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-05-03)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- GA release
```

Comment:
If there are changes to the public surface area of the SDK, such as model renames or changes to properties, we should call those out here for customers to refer to when upgrading from the beta to the stable version of the library.

Reply:
OK,  I will list all changes

Reply:
I listed all relevant changes in CHANGELOG.md